libnmstate/netinfo.py

```diff
@@ -15,6 +15,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 from __future__ import absolute_import
+from operator import itemgetter
```

Comment:
I never got the point of using a helper like this instead of a lambda. Feels like wasting an import line.

Reply:
https://stackoverflow.com/questions/11287207/why-should-i-use-operator-itemgetterx-instead-of-x

And pylint will complain if you use lambda instead of itemgetter.